examples/stable_diffusion_xl/inference_lcm.md

```diff
@@ -0,0 +1,74 @@
+# LCM Sampler for Stable Diffusion XL (SDXL)
+
+[Latent Consistency Models: Synthesizing High-Resolution Images with Few-Step Inference](https://arxiv.org/abs/2310.04378)
+
+Latent Consistency Models (LCM) enable quality image generation in typically 2-4 steps making it possible to use diffusion models in almost real-time settings.
+
+## Dependency
+
+- mindspore 2.2
+- openmpi 4.0.3 (for distributed mode)
+
+To install the dependency, please run
+
+```shell
+pip install -r requirements.txt
+```
+
+## Preparation
+
+### Convert Pretrained Checkpoint
```

Comment:
Why do we need to convert the SDXL checkpoint again?  I thought it's already done previously for stability-ai SDXL -> ms SDXL. 

For diffusers SDXL, it's actually converted from  stability-ai SDXL. Re-using existing current ms SDXL checkpoint should be fine?

Reply:
I tried to directly use sd_xl_base_1.0.safetensors+lcm-sdxl to generate ms SDXL, but there were issues with the generated results. I also experimented with torch, and some strange pictures were also generated. The result of using vae+text_encoder+text_encoder_2 in sdxl and lcm-sdxl is currently the best.